doc/xml/daps_user_output.xml

```diff
@@ -277,4 +277,34 @@
    </para>
   </important>
  </sect1>
-</chapter>
+ <sect1 id="sec.daps.user.output.timestamp">
+ <title>Putting the actual date and time into the target document</title>
+ <para>
+ In order to get information about the build date and time, you can use the dbtimestamp tag.
+ The dbtimestamp tag is a so called processing instruction, so it will be processed by Daps, when the document
+ is built and the date and time will show up in the final document, based on how you've set the date/time format.
+ </para>
+ <variablelist>
```

Comment:
I have two issues:

* Replace `variablelist` with an `example` element (that is made for). :wink: 
* The term contains "tag" which is wrong; it's a PI (processing instruction)


```xml
<example>
   <title>Inserting the <sgmltag class="pi">dbtimestamp</sgmltag>PI</title>
    <screen>&lt;para>Current date: &lt;?dbtimestamp?>.&lt;/para></screen>
</example>
```